serialization/core/tests/simple_types.rs

```diff
@@ -201,7 +201,7 @@ fn test_scale_options() {
     assert_eq!(dec, Some(OptionWrapper::new(result)));
 
     // Decode and encode 1_048_576 chars 'X'
-    let result = Some(format!("!{:X<4194304}!", ""));
+    let result = Some(std::iter::repeat_n('X', 1_048_576).collect::<String>());
```

Comment:
FYI: this was a hard error (probably since 1.87, the version I've skipped) - `integer out of range for u16 in format string`